doc/notifications.md

```diff
@@ -135,21 +212,41 @@ Follow the [instructions](https://core.telegram.org/bots#6-botfather) to set up
 Message the [GetID bot](https://t.me/getidsbot) to find your chat ID.
 Multiple chat IDs can be provided in order to deliver notifications to multiple recipients.
 
+### Configuration file
+
 * `telegram`
   * `token`: Telegram bot token. **required**
   * `chat_ids`: List of chat IDs to send notifications to. **required**
 
+### Environment variables
+
+* `DIUN_NOTIF_TELEGRAM_TOKEN`
+* `DIUN_NOTIF_TELEGRAM_CHATIDS`
```

Comment:
How is a list of chat IDs represented when stored in an environment variable? From reading this, it is not obvious for me without looking at the code.

```
DIUN_NOTIF_TELEGRAM_CHATIDS="1,2,3,4,5"
DIUN_NOTIF_TELEGRAM_CHATIDS="1 2 3 4 5"
...
```

Reply:
@Tooa Yeah, it's kind of sketchy at the moment. I'm working on it. But it will be separated by a comma indeed.

Reply:
See https://github.com/crazy-max/diun/pull/82/files#diff-6c01764051bc3df7cf9a13dbf7fb58d4R220